eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -78,12 +78,7 @@ public abstract class PhDefault implements Phi, Cloneable {
     /**
      * Terms being processed now.
      */
-    private final ThreadLocal<Set<Integer>> terms = new ThreadLocal<>();
-
-    /**
-     * Terms being processed now.
-     */
-    private final ThreadLocal<Set<Integer>> strings = new ThreadLocal<>();
+    private final static ThreadLocal<Set<Integer>> terms = new ThreadLocal<>();
```

Comment:
@mximp I think the right name should be `TERMS`, since it's static

Reply:
fixed